docs/resources/group_eligible_assignment.md

```diff
@@ -0,0 +1,32 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "azurepim_group_eligible_assignment Resource - terraform-provider-azurepim"
+subcategory: ""
+description: |-
+  Manages an Entra Group PIM Eligible Role Assignment.
+---
+
+# azurepim_group_eligible_assignment (Resource)
+
+Manages an Entra Group PIM Eligible Role Assignment.
```

Comment:
```suggestion
Enables PIM for an Entra group, manages an PIM Eligible Role Assignment and sets the PIM policy for the member role to allow for no expiration on eligible assignments.  
```

I'd like to mention this one

Reply:
Added this to the go code where this markdown is generated from.